detection-rules/link_fake_fax_low_reputation.yml

```diff
@@ -0,0 +1,82 @@
+name: "Brand Impersonation: Fake fax with low reputation link"
+description: |
+  Detects messages with low reputation links, presence of brand logos and mentions of "fax" in subject, sender or body from a first-time sender. 
+references:
+  - "https://www.hoxhunt.com/blog/fax-phishing"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and 0 < length(body.links) < 5
+  and sender.email.domain.root_domain not in $org_domains
+  and any(body.links,
+      (
+          .href_url.domain.domain not in $tranco_1m or
+          .href_url.domain.domain in $free_file_hosts or
+          .href_url.domain.root_domain in $free_subdomain_hosts or
+          .href_url.domain.domain in $url_shorteners or
+  
+          // mass mailer link, masks the actual URL
+          .href_url.domain.root_domain in (
+              "hubspotlinks.com",
+              "mandrillapp.com",
+              "sendgrid.net",
+          )
+      )
+  )
+  
+  // any brand logo detected
+  and (
+      any(attachments,
+          .file_type in ('png', 'jpeg', 'jpg', 'bmp')
+          and any(ml.logo_detect(.).brands, .name is not null)
+      )
+      or any(ml.logo_detect(beta.message_screenshot()).brands,
+          .name is not null)
```

Comment:
do we need to exclude legit senders for the supported brands that might be impersonated? and/or want to look for high confidence? it seems a bit broad otherwise due to the tranco/mass mailer checks, but i'll defer to you on whether either of these are necessary